src/main/java/jalau/at18/architects/Wonders/StagesHal.java

```diff
@@ -0,0 +1,26 @@
+package jalau.at18.architects.Wonders;
+enum StagesHal {
```

Comment:
You probably meant "StagesHall"?

Reply:
no, it is an abbreviation of "Halicarnassus", as the name was too long.